tests/integration/records/directory_validator.py

```diff
@@ -61,7 +61,20 @@ def validate_records_schema(self):
                     'integer', 'string', 'string', 'string',
                     'string', 'string', 'string', 'date', 'string',
                     'datetime', 'datetimetz'
-                ]
+                ],
+                # MySQL's datetimetz type ("TIMESTAMP") doesn't
+                # support dates before the Unix epoch (Jan 1 1970),
+                # and records-mover does not yet support using
+                # inference to determine if the data in question will
+                # fit into it.
+                #
+                # https://app.asana.com/0/1128138765527694/1166526213569051
+                # https://stackoverflow.com/questions/31761047/what-difference-between-the-date-time-datetime-and-timestamp-types/56138746
+                [
+                    'integer', 'string', 'string', 'string',
+                    'string', 'string', 'string', 'date', 'time',
+                    'datetime', 'datetime'
+                ],
             ]
             assert actual_field_types in acceptable_field_types,\
```

Comment:
Should this check be tied actually to the database it is for? Like, it would be a bug if Redshift did what mysql did, right? Or are we verifying that some other way?

Reply:
I took a crack at strengthening this verification in https://github.com/bluelabsio/records-mover/pull/39